app/src/main.cpp

```diff
@@ -226,51 +254,33 @@ namespace scitt
             "Failed to get host time: {}", ccf::api_result_to_str(result)));
         }
 
-        SCITT_DEBUG("Get SCITT configuration from KV store");
+        SCITT_DEBUG("Get service configuration from KV store");
         auto cfg = ctx.tx.template ro<ConfigurationTable>(CONFIGURATION_TABLE)
                      ->get()
                      .value_or(Configuration{});
 
-        ClaimProfile claim_profile;
+        SignedStatementProfile signed_statement_profile;
         cose::ProtectedHeader phdr;
         cose::UnprotectedHeader uhdr;
         try
         {
-          SCITT_DEBUG("Verify submitted claim");
-          std::tie(claim_profile, phdr, uhdr) = verifier->verify_claim(
-            body, ctx.tx, host_time, DID_RESOLUTION_CACHE_EXPIRY, cfg);
+          SCITT_DEBUG("Verify submitted signed statement");
+          std::tie(signed_statement_profile, phdr, uhdr) =
+            verifier->verify_signed_statement(
+              body, ctx.tx, host_time, DID_RESOLUTION_CACHE_EXPIRY, cfg);
```

Comment:
if DID:WEB verification is going away why do we still have `DID_RESOLUTION_CACHE_EXPIRY`?